### DISCUSSION TITLE
Eroding Contact Simulation of a Projectile impacting Plate

### DISCUSSION THREAD

**Discussion by CAE4U** [OpenRadioss/OpenRadioss#3114] [Show and tell]
Hello OpenRadioss users,
We are glad to inform that, after quite a bit of exploring (and still exploring :-)) OpenRadioss, we were able to perform a Eroding Contact Simulation of a a Projectile impacting Plate.
It was not easy, but really a great pleasure while looking at the results.
We appreciate any of your insights for further improvements.

You can use the below link to watch our work,
https://youtu.be/779NdtBwnDU

Best Regards,
CAE4U

**Comment by MarianBulla** [comment]
Hello @CAE4U ,
this is really great work !!!

Yes, it is not easy, but it is also a very complex topic, but you did it !
So, congratulations for you great work, demonstrating the capabilities of OpenRadioss.

You can review the presentation from Paul Du Bois, showing the roots of OpenRadioss here: https://openradioss.org/presentations-aachen270623/
(especially at time 2:30 min.)

Therefore, it is more understnadable, that it will deliver almost the same results as the other commercial solver :-)

Again, thank you very much for your interest using OpenRadioss.

Thanks and best regards and looking forward for more from you,
Marian

**Reply by CAE4U** [reply]
Hello Marian,
Thank you for your appreciation, it was really motivating.

We would like to appreciate the reference. It is insightful and surprising, that it reverberated our very short experience of using OpenRadioss and other commercial code (discontinued). While is was nice to know about the roots of the code, the subsequent section on convergence was breathtaking. It consoled us that we are not alone when it comes to validation :-) using failure models and it is historical.

We do hope more of these kind of complex examples can be demonstrated along with its validation.
However from our (short) experience, following a true Open-Source Chain for meshing and setup is a daunting task. Limited hardware adds up to the challenge. 

Any ways we will try our best. Keep Supporting us.

You may want to showcase our work as examples.

We hope you have our channel in your subscription list, so that you get notified on our work even before we show case in the forum.

Best Regards,
CAE4U

**Comment by RevolT87** [comment]
Hi @CAE4U,
This is really great work indeed !!
Could you share the models so we can see if we could suggest some improvements? 
You may use quality criteria in the solid property to remove elements in case the quality decrease
Also, you may use sol2sph to replace solid elements to sph in order not to loose any mass. This will be however more costly.
Regards,
Mathis.

**Reply by CAE4U** [reply]
Hi Mathis,

Glad to know you liked our work.

All the files including the models can be obtained from https://cae4u.gumroad.com/l/xumbu, (an opportunity to help us :-)).

It is interesting to know about the improvements. In this regard we seek some clarifications.

Could you please refer some documentation on how the element removal on quality-decrease works?

We believe, even while using commercial codes and as in the reference, lot of non-intuitive manipulations of the material model seems to be used. This makes the numerical model more specific to the case but not even to the class of the problem. The only objective appears to be matching the results, but not building a reliable numerical model which can be used across the class of the problem. Further, such models cannot be reliably used for new designs. We have touched up on these lines in our video. 

We faced very strange issues with tried with material data as specified in the reference. 

Can we see optimizing the material constants as a better solution? Is there any option in the tool to feed the material stress strain curves,run some optimization to get the best possible co-efficient? If not, can we expect in future? This can provide a intuitive solution to deploy with new design assessment.

Further,We are aware of using SPH method, we have very limited computational power. So we have challenges using this approach. Even for FEA we used coarse model. The simulation time is significantly large even for modest refinements.


Best Regards,
CAE4U
